presto-server-rpm/pom.xml

```diff
@@ -33,143 +33,159 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
It's hard to see on github, but this part is basically unchanged.  I just took it out from being inside a profile. The only part here that is changed is the version of redlinerpm-maven-plugin to 2.0